catalog/core/catalog-core-standardframework/src/main/resources/OSGI-INF/metatype/downloadManager.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License as published by the Free Software Foundation, either
+ * version 3 of the License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+
+ -->
+<metatype:MetaData xmlns:metatype="http://www.osgi.org/xmlns/metatype/v1.0.0">
+
+    <OCD description="Resource Download Configuration"
+         name="Resource Download Settings"
+         id="ddf.catalog.resource.download.ReliableResourceDownloadManager">
+        <AD name="Product Cache Directory" id="productCacheDirectory" required="false"
+            type="String" default=""
+            description="Directory where retrieved products will be cached for faster, future retrieval.
+             If a directory path is specified with directories that do not exist,
+             Catalog Framework will attempt to create those directories.
+             Out of the box (without configuration), the product cache directory is
+             INSTALL_DIR/data/product-cache. If a relative path is provided it will be relative
+             to the INSTALL_DIR. It is recommended to enter an absolute directory path such as
+             /opt/product-cache in Linux or C:/product-cache in Windows."/>
+        <AD name="Enable Product Caching" id="cacheEnabled" required="false" type="Boolean"
+            default="true"
+            description="Check to enable caching of retrieved products."/>
+        <AD name="Delay (in seconds) between product retrieval retry attempts"
+            id="delayBetweenRetryAttempts" required="false"
+            type="Integer" default="10"
+            description="The time to wait (in seconds) between attempting to retry retrieving a product."/>
+        <AD name="Max product retrieval retry attempts" id="maxRetryAttempts" required="false"
+            type="Integer" default="3"
+            description="The maximum number of attempts to retry retrieving a product."/>
+        <AD name="Product Retrieval Monitor Period" id="retrievalMonitorPeriod" required="false"
+            type="Integer" default="5"
+            description="How many seconds to wait and not receive product data before retrying to retrieve a product."/>
+        <AD name="Always Cache Product" id="cacheWhenCanceled" required="false" type="Boolean"
+            default="false"
+            description="Check to enable caching of retrieved products even if client cancels the download."/>
```

Comment:
I would add to the description that this flag has no effect if _Enable Product Caching_ is disabled.